### PR TITLE
Notify user of new keptn/bridge version

### PR DESCRIPTION
**Pull request by ermin-muratovic**
This feature allows users to enable automatic version check.

When the new bridge with this feature is used for the very first time, it will show a confirmation dialog, informing the user about that feature.

first-time

Once that feature is enabled, it will compare the current version with available versions in [version.json](https://raw.githubusercontent.com/keptn/get.keptn.sh/master/src/version.json) and show a notification on top of the bridge, when a new version (either stable or prerelease) is available.

notification-EAP

Of course the automatic version check can be disabled in the user menu on the top right.

### CODE REVIEW

File: bridge/client/app/app-header/app-header.component.ts [hunk 1]
```diff
@@ -33,10 +44,80 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
...
+    });
+  }
+
+  doVersionCheck(currentVersion, stableVersions, prereleaseVersions, type) {
+    stableVersions.forEach(stableVersion => {
+      if(semver.lt(currentVersion, stableVersion)) {
+        let genMessage;
+        switch(semver.diff(currentVersion, stableVersion)) {
+          case 'patch':
+            genMessage = (version, type, major, minor) => `New ${type} ${version} available. This is a patch with bug fixes for your current version. For details how to upgrade visit https://keptn.sh/docs/${major}.${minor}.0/installation/upgrade-keptn`;
```
Comment (johannes-b) [inline bridge/client/app/app-header/app-header.component.ts:75]: Can you please change the link to: `https://keptn.sh/docs/${major}.${minor}.0/operate/upgrade/`

File: bridge/client/app/app-header/app-header.component.ts [hunk 2]
```diff
@@ -33,10 +44,80 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
...
+            break;
+          case 'minor':
+            genMessage = (version, type, major, minor) => `New ${type} ${version} available. This is a minor update with backwards compatible improvements. For details how to upgrade visit https://keptn.sh/docs/${major}.${minor}.0/installation/upgrade-keptn`;
```
Comment (johannes-b) [inline bridge/client/app/app-header/app-header.component.ts:78]: Can you please change the link to: `https://keptn.sh/docs/${major}.${minor}.0/operate/upgrade/`

File: bridge/client/app/app-header/app-header.component.ts [hunk 3]
```diff
@@ -33,10 +44,80 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
...
+            break;
+          case 'major':
+            genMessage = (version, type, major, minor) => `New ${type} ${version} available. This is a major update and it might contain incompatible changes. For details how to upgrade visit https://keptn.sh/docs/${major}.${minor}.0/installation/upgrade-keptn`;
```
Comment (johannes-b) [inline bridge/client/app/app-header/app-header.component.ts:81]: Can you please change the link to: `https://keptn.sh/docs/${major}.${minor}.0/operate/upgrade/`

File: bridge/client/app/app-header/app-header.component.ts (hunk 1 above)
Comment (johannes-b) [inline bridge/client/app/app-header/app-header.component.ts:75]: Can you please change the link to: `https://keptn.sh/docs/${major}.${minor}.0/operate/upgrade/`

File: bridge/client/app/app-header/app-header.component.ts [hunk 4]
```diff
@@ -33,10 +44,80 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
...
+        }
+
+        let major = semver.major(stableVersion);
+        let minor = semver.minor(stableVersion);
+        this.notificationsService.addNotification(NotificationType.Info, genMessage(stableVersion, type, major, minor));
+      }
+    });
+    prereleaseVersions.forEach(prereleaseVersion => {
+      if(semver.lt(currentVersion, prereleaseVersion)) {
+        let genMessage = (version, type, major, minor) => `New ${type} ${version} available in Early Access. This is a prerelease version and should be treated with care. For details how to upgrade visit https://keptn.sh/docs/${major}.${minor}.0/installation/upgrade-keptn`;
```
Comment (johannes-b) [inline bridge/client/app/app-header/app-header.component.ts:92]: Can you please change the message to: `New ${type} ${version} available. This is a pre-release version that might contain experimental features. For details how to upgrade visit: https://keptn.sh/docs/${major}.${minor}.0/operate/upgrade/`

File: bridge/client/app/app-header/app-header.component.ts (hunk 1 above)
Comment (ermin-muratovic) [reply bridge/client/app/app-header/app-header.component.ts]: Has been changed ;)

File: bridge/client/app/app-header/app-header.component.ts (hunk 4 above)
Comment (ermin-muratovic) [reply bridge/client/app/app-header/app-header.component.ts]: Thank you @johannes-b 
I changed the message for the pre-release version. Have you also reviewed the other messages?

**Pull request merged**